.github/actions/aws-ci/action.yml

```
@@ -14,7 +14,13 @@ inputs:
     description: 'AWS account type'
     required: false
     default: single-account
-
+  cloudbeat-config:
+    description: 'Full path to a cloudbeat configuration file, e.g. "deploy/aws/cloudbeat-aws.yml"'
+    required: true
+  pytest-keyword-expression:
+    description: 'Keyword expression to use as a pytest -k flag valyu'
```

```suggestion
    description: 'Keyword expression to use as a pytest -k flag value'
```

Maybe add what values are expected? `aws` and the one specific to asset inventory?

---

Resolved by having a separate action for asset inventory.